src/client/components/Footer/index.js

```diff
@@ -1,17 +1,25 @@
+import PropTypes from "prop-types";
 import Container from "react-bootstrap/Container";
 import Nav from "react-bootstrap/Nav";
 import Navbar from "react-bootstrap/Navbar";
 import React from "react";
 import { useTranslation } from "react-i18next";
 
-function Footer() {
+function Footer(props) {
   const { t } = useTranslation();
 
+  const basePathName = window.location.pathname;
+  const backToTopPathName = basePathName.includes("#")
```

Comment:
I don't think location.pathname includes #. That's in location.hash, so you don't need this conditional.

Reply:
Good catch! Will add a ticket to clean up later!